components/auth/Verify.tsx

```diff
@@ -111,12 +111,12 @@ export const Verify = ({ username, authenticationFlowToken }: VerifyProps): Reac
                 >
                   {t("verify.fields.confirmationCode.label")}
                 </Label>
-                <Description className="text-p text-black-default" id={"verificationCode-hint"}>
```

Comment:
Slowly dropping the Description component to be more consistent. The Description component is specific to forms needs and the app needs more flexibility sometimes e.g. having a bullet list
This could be solved by moving e.g. the bullet list outside the Description but then a container div would be needed to setup the aria-describes reference. More work for no benefit